src/main/java/com/azure/cosmos/examples/diagnostics/sync/CosmosDiagnosticsQuickStart.java

```diff
@@ -0,0 +1,258 @@
+        });
+
+        logger.info("Done.");
+    }
+
+    private void replaceDocument() throws Exception {
+        logger.info("Replace document " + documentId);
+
+        // Replace existing document with new modified document
+        Family family = new Family();
+        family.setLastName(documentLastName);
+        family.setId(documentId);
+        family.setDistrict("Columbia"); // Document modification
+
+        CosmosItemResponse<Family> itemResponse =
+            container.replaceItem(family, family.getId(), new PartitionKey(family.getLastName()),
+                new CosmosItemRequestOptions());
+
+        CosmosDiagnostics diagnostics = itemResponse.getDiagnostics();
+        logger.info("Replace item diagnostics : {}", diagnostics);
+
+        logger.info("Request charge of replace operation: {} RU", itemResponse.getRequestCharge());
+
+        logger.info("Done.");
+    }
+
+    private void upsertDocument() throws Exception {
+        logger.info("Replace document " + documentId);
+
+        // Replace existing document with new modified document (contingent on modification).
+        Family family = new Family();
+        family.setLastName(documentLastName);
+        family.setId(documentId);
+        family.setDistrict("Columbia"); // Document modification
+
+        CosmosItemResponse<Family> itemResponse =
+            container.upsertItem(family, new CosmosItemRequestOptions());
+
+        CosmosDiagnostics diagnostics = itemResponse.getDiagnostics();
+        logger.info("Upsert item diagnostics : {}", diagnostics);
+
+        logger.info("Done.");
+    }
+
+    // Document delete
+    private void deleteDocument() throws Exception {
+        logger.info("Delete document " + documentId + " by ID.");
+
+        // Delete document
+        CosmosItemResponse<Object> itemResponse = container.deleteItem(documentId,
+            new PartitionKey(documentLastName), new CosmosItemRequestOptions());
+
+        CosmosDiagnostics diagnostics = itemResponse.getDiagnostics();
+        logger.info("Delete item diagnostics : {}", diagnostics);
+
+        logger.info("Done.");
+    }
+
+    // Database delete
+    private void deleteDatabase() throws Exception {
+        logger.info("Last step: delete database " + databaseName + " by ID.");
+
+        // Delete database
+        CosmosDatabaseResponse dbResp = client.getDatabase(databaseName).delete(new CosmosDatabaseRequestOptions());
+        logger.info("Status code for database delete: {}", dbResp.getStatusCode());
+
+        logger.info("Done.");
+    }
+
+    // Cleanup before close
+    private void shutdown() {
+        try {
+            //Clean shutdown
+            deleteDocument();
+            deleteDatabase();
+        } catch (Exception err) {
+            logger.error("Deleting Cosmos DB resources failed, will still attempt to close the client. See stack "
+                + "trace below.");
+            err.printStackTrace();
```

Comment:
we should use logger for exception and not use printSTacktrace.

Reply:
not sure how this one got missed, will update it.